tests/check-uefi.c

```diff
@@ -263,7 +271,7 @@ START_TEST(bootman_uefi_remove_bootloader)
         autofree(BootManager) *m = NULL;
 
         fail_if(!nc_file_exists(TOP_BUILD_DIR "/tests/update_playground/" BOOT_DIRECTORY
-                                              "/EFI/Boot"),
+                                              "/efi/BOOT"),
```

Comment:
This is not the original case.

Reply:
[this line in v1.5.5](https://github.com/clearlinux/clr-boot-manager/blob/v1.5.5/tests/check-uefi.c#L266)

Maybe I misunderstand, what do you refer to as original case?

Reply:
At this point its really hard to tell - so I apologize if I'm wrong on some of these. My primary concern is restoring sync between the old and new test cases. That and ensuring we're using nc_ case function for all paths on an ESP (even for joining paths! :))